astropy/table/table.py

```diff
@@ -1218,7 +1218,6 @@ def _init_from_list_of_dicts(self, data, names, dtype, n_cols, copy):
 
                 # Finally do the masking in a mixin-safe way.
                 self[name][indexes] = np.ma.masked
-        return
```

Comment:
@taldcroft I don't think these `return` and `return None` are doing anything as functions return `None` by default. However, please comment if you think these should be left in for code clarity.